ros/src/computing/perception/localization/packages/ndt_localizer/nodes/ndt_matching/ndt_matching.cpp

```diff
@@ -534,11 +528,15 @@ static void gnss_callback(const geometry_msgs::PoseStamped::ConstPtr& input)
   tf::Quaternion gnss_q(input->pose.orientation.x, input->pose.orientation.y, input->pose.orientation.z,
                         input->pose.orientation.w);
   tf::Matrix3x3 gnss_m(gnss_q);
+
+  pose current_gnss_pose;
   current_gnss_pose.x = input->pose.position.x;
   current_gnss_pose.y = input->pose.position.y;
   current_gnss_pose.z = input->pose.position.z;
   gnss_m.getRPY(current_gnss_pose.roll, current_gnss_pose.pitch, current_gnss_pose.yaw);
 
+  static pose previous_gnss_pose = current_gnss_pose;
```

Comment:
Why previous_gnss_pose = current_gnss_pose here? offset_x/y/z will be 0 at line 558, 559, 560.

Reply:
of course, offset_x/y/z is 0 at first step. 
But previous_gnss_pose is static variable. so next step, line 538 will be ignored.